tss-esapi/src/context/handle_manager.rs

```diff
@@ -41,9 +41,14 @@ impl HandleManager {
         }
 
         if self.open_handles.contains_key(&handle) {
-            error!("Handle({}) is already open", ESYS_TR::from(handle));
-            return Err(Error::local_error(WrapperErrorKind::InvalidHandleState));
+            // It is safe to call unwrap because the existance of the key has already been verified.
+            let stored_handle_drop_action = self.open_handles.get(&handle).unwrap();
```

Comment:
It's possible to avoid the unwrap by using `if let` just like in the other PR.

I trust you'll adjust it and I don't want to unnecessarily block the workflow so I'll ack the PR now :)